src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/IJobs.java

```diff
@@ -0,0 +1,10 @@
+package io.jenkins.plugins.report.jtreg.main.comparator;
+
+import java.io.File;
+import java.util.ArrayList;
+
+public interface IJobs {
+    ArrayList<File> getJobs();
+    void printJobs(boolean skipFailed, String nvrQuery, int numberOfBuilds);
```

Comment:
I doubt a bit the print jobs  should be there.  This thing should slimply provides jobs. Lukily, nvr is seaprate switch, so *additional* filtering of jobs. or not?